### ISSUE TITLE
Add multiple account support

### ISSUE THREAD

**Issue by gurland** [gurland/telegram-delete-all-messages#39]
Implement ability to work with multiple accounts.

For CLI interactive mode we have to:
- Remember last successful login and display that session as default
- Display list of all available sessions
- Let the user to add/remove sessions

For default argument based CLI (should be implemented in #11) simply add argument with name/path to client session file.

Sessions in pyrogram are stored in `.session` files which are simply SQlite database files. https://docs.pyrogram.org/topics/storage-engines.

**Comment by xanoni** [comment]
As a workaround, I just have a bash script swap out the `Config.session` file before I invoke the program.

**Comment by gurland** [comment]
I agree, that there is no need to overwhelm simple interface we have with multiple accounts
